packages/react-cards/src/components/Card/CardSection/CardSection.view.tsx

```diff
@@ -0,0 +1,19 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { withSlots, getSlots } from '@uifabric/foundation';
+import { Stack } from 'office-ui-fabric-react';
+import { ICardSectionComponent, ICardSectionProps, ICardSectionSlots } from './CardSection.types';
+
+export const CardSectionView: ICardSectionComponent['view'] = props => {
+  const { children, ...rest } = props;
+  if (React.Children.count(children) < 1) {
+    return null;
+  }
+
+  const Slots = getSlots<ICardSectionProps, ICardSectionSlots>(props, {
+    root: Stack,
+    inner: 'div'
```

Comment:
Is this used?

Reply:
`inner` is not used but `getSlots` requires you to define each slot in specified by the `interface`. I'll change this so that `ICardSectionSlots` extends from `Pick<IStackSlots, 'root'>` instead of from `IStackSlots`, then we can remove `inner` from here.